adagucserverEC/CRequest.cpp

```diff
@@ -1823,7 +1823,56 @@ int CRequest::process_all_layers() {
       // dataSources[j]->getCDFDims()->addDimension("none","0",0);
     }
     if (dataSources[j]->dLayerType == CConfigReaderLayerTypeLiveUpdate) {
+      // This layer has no dimensions, but we need to add one timestep with data in order to make the next code work.
       layerTypeLiveUpdateConfigureDimensionsInDataSource(dataSources[j]);
+      if (dataSources[j]->cfgLayer->DataPostProc.empty()) {
```

Comment:
What does this condition mean? Is it only for the solar terminator post proc?

Reply:
This is the condition where we differentiate between the default liveupdate layer, and a complex layer with post processors (that is, the solar terminator). If you enter the CImageDataWriter code with the default liveupdate, the code breaks in many places. I tried to fix it but as I went deeper it got worse.